wiki/Main_Page/id.md

```diff
@@ -1,46 +1,45 @@
 ---
-layout: main_page
-outdated: true
+layout: "main_page"
 ---
 
+<!-- Do not add any empty lines inside this div. -->
 <div class="wiki-main-page__blurb">
-osu! wiki adalah sebuah projek berdasarkan orang-orang yang berkontribusi dalam gim osu!. Tujuan akhirnya adalah menggantikan bagian FAQ yang lama, dan memungkinkannya menjadi sumber yang jauh lebih lengkap. Namun, untuk memungkinkan ini, kami membutuhkan bantuan dari semua orang yang mau berkontribusi!
-<br>
-Kamu dapat menggunakan akun osu! kamu untuk mulai berkontribusi!
+Welcome to the osu! wiki, a project containing a wide range of osu! related information.
+Selamat datang di osu! wiki, sebuah proyek yang berisi berbagai informasi terkait osu!
 </div>
 
 
 <div class="wiki-main-page__panels">
-<div class="wiki-main-page-panel wiki-main-page-panel--full">
+<div class="wiki-main-page-panel">
 
 
-# Mulai Permain
+# Memulai
 
-[Selamat Datang](/wiki/Welcome) • [Pendaftaran](/wiki/Registration) • [Instalisasi](/wiki/Installation) • [Antarmuka Permainan](/wiki/Interface) • [Opsi Permainan](/wiki/Options) • [Referensi Tombol Pintasan](/wiki/Shortcut_Key_Reference) • [Peta Wiki](/wiki/Sitemap) • [Panduan](/wiki/Guides) • [Layanan Bantuan](/wiki/Help_Centre)
+[Selamat Datang](/wiki/Welcome) • [Registrasi](/wiki/Registration) • [Instalasi](/wiki/Installation) • [Antarmuka Game](/wiki/Interface) • [Pengaturan Game](/wiki/Options) • [Referensi Tombol Pintasan](/wiki/Shortcut_Key_Reference) • [Sitemap](/wiki/Sitemap) • [Panduan](/wiki/Guides) • [Pusat Bantuan](/wiki/Help_Centre)
 
 </div>
 <div class="wiki-main-page-panel">
 
 
 # Gameplay
 
-[FAQ](/wiki/FAQ) • [Multi](/wiki/Multi) • [Mode Permainan](/wiki/Game_Modes): ([osu!](/wiki/Game_Modes/osu!) • [osu!taiko](/wiki/Game_Modes/osu!taiko) • [osu!catch](/wiki/Game_Modes/osu!catch) • [osu!mania](/wiki/Game_Modes/osu!mania))
+[FAQ](/wiki/FAQ) • [Multi](/wiki/Multi) • [Mode Game](/wiki/Game_Modes): ([osu!](/wiki/Game_Modes/osu!) • [osu!taiko](/wiki/Game_Modes/osu!taiko) • [osu!catch](/wiki/Game_Modes/osu!catch) • [osu!mania](/wiki/Game_Modes/osu!mania))
 
 </div>
 <div class="wiki-main-page-panel">
 
 
-# [Menyuting Beatmap](/wiki/Beatmap_Editor)
+# [Editor Beatmap](/wiki/Beatmap_Editor)
```

Comment:
Refers to line 40,
In line 40, we are using **Penyuntingan** meanwhile in here we are using **Editor**
Just some advice to change with **Penyunting Beatmap**, Ignore it if Beatmap Editor is Special Terms in osu!